Makefile.am

```diff
@@ -81,6 +81,7 @@ $(BITCOIN_WIN_INSTALLER): all-recursive
 	  echo error: could not build $@
 	@echo built $@
 
```

Comment:
Why is this change necessary for the 0.5 branch? In addition, the following PR indicates that changes to this file alone are not sufficient.

https://github.com/bitcoin/bitcoin/pull/28432

Reply:
I realise what happened. The CI had completed because it skipped the packaging step. I was so relieved about its success that it did not even occur to me to check the logs. 
There are gaps in macOS packaging. I'm fixing them.